tests/test_logging.py

```diff
@@ -178,59 +178,104 @@ def setup_logging_config(cli_options: Options, file_option_value: Optional[str]
         # Reset stream buffer
         stream.truncate()
 
+    def get_output_and_clear() -> str:
+        value = stream.getvalue()
+        stream.seek(0)
+        stream.truncate(0)
+        return value
+
+    # Log file to pass in the options.
+    # This config option will be overridden by the other config in this test case.
+    old_log_file = os.path.join(tmpdir, "old_log_file.txt")
     path_logging_config_file1 = os.path.join(tmpdir, "logging_config1.yml")
     path_logging_config_file2 = os.path.join(tmpdir, "logging_config2.yml")
     path_logging_config_file3 = os.path.join(tmpdir, "logging_config3.yml")
 
     # Set --logging-config option on CLI only
     write_logging_config_file(path=path_logging_config_file1, formatter="AAA %(message)s")
     # Also assert that the other config options are ignored when logging_config is set.
-    setup_logging_config(cli_options=Options(logging_config=path_logging_config_file1, verbose=1), file_option_value=None)
+    options1 = {"log_file": old_log_file, "log_file_level": "4", "keep_logger_names": False}
```

Comment:
I think we need to provide a better warning to the user than a dictionary of an internal data structure. Why not just show the actual options names, i.e. `--log-file`, `--log-file-level`, etc.